site/content/docs/5.0/components/alerts.md

```diff
@@ -147,35 +147,42 @@ Loop that generates the modifier classes with the `alert-variant()` mixin.
 
 ## JavaScript behavior
 
-### Triggers
+### Initialize
 
-Enable dismissal of an alert via JavaScript:
+Initialize elements as alerts
 
 ```js
 var alertList = document.querySelectorAll('.alert')
+var alerts = []
```

Comment:
I'm not sure about this here, maybe we should use `map`? Just thinking out loud here.